packages/graphiql-plugin-utils/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@graphiql/plugin-utils",
+  "description": "shared tooling for building graphiql plugins",
+  "version": "0.0.1",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "cjs/vite-plugin.js",
```

Comment:
why do you bundle to cjs? you just wanted to use esm only everywhere is possible

Reply:
our users might still use cjs for vite. this is meant for more than the readme, see the docs. publishing esm and using esm are different things

Reply:
@B2o5T please understand the difference. the retire cjs branch only aims to publish esm, but it will take a larger effort to convert the whole monreopo to esm. i have been trying to illustrate the difference for days. that is planned for later, please see the comment in `retire-js` where i talk about converting to type module as a next step. we don't need to do this all in one PR, it would be way too much to review at once

Reply:
vite and webpack and the examples are handling the esm only exports just fine. that is different from converting the whole monorepo to use type: "module", which we may not need to do yet

Reply:
ok, I just saw that vite plugin react and vite plugin dts exports cjs and mjs, let's keep